modules/assim.sequential/R/sample.parameters.R

```diff
@@ -1,6 +1,6 @@
 ##' @name sample.parameters
 ##' @title sample parameters
-##' @author Michael Dietze \email{dietze@bu.edu}
+##' @author Michael Dietze \email{dietze@@bu.edu}
```

Comment:
@mdietze note that these emails created invalid documentation files (need to use `mdietze@@bu` instead of `mdietze@bu` http://stackoverflow.com/a/8809391/199217